### DISCUSSION TITLE
在不设置红脸出击的情况下，红脸了会直接停止脚本

### DISCUSSION THREAD

**Discussion by QuincyOvO** [LmeSzinc/AzurLaneAutoScript#974] [Q&A] [answered]
在不设置红脸出击的情况下，红脸了会直接停止脚本，不会反悔港口等待或者进入下一任务
请问这是我设置错了还是本身就不支持这个功能呢？

**Comment by LmeSzinc** [comment, marked as answer]
1. 确认 Alas 内的设置与游戏内的情况对应，比如你在 Alas 里设置了后宅二楼，那么游戏里就应该放置在后宅二楼，以回复最慢的角色为准。
2. 确认你的队伍强度足够，不会沉船。
3. 不要在多个任务之间共享同一角色，因为不同任务之间的心情计算是独立的
4. 不要更换队伍中的角色，如果你真的换了，需要手动修改 Alas 里的心情值和记录时间

**Reply by Qilingwan** [reply]
> 1. 确认 Alas 内的设置与游戏内的情况对应，比如你在 Alas 里设置了后宅二楼，那么游戏里就应该放置在后宅二楼，以回复最慢的角色为准。
> 2. 确认你的队伍强度足够，不会沉船。
> 3. 不要在多个任务之间共享同一角色，因为不同任务之间的心情计算是独立的
> 4. 不要更换队伍中的角色，如果你真的换了，需要手动修改 Alas 里的心情值和记录时间

管用
